tutorials/setup.md

```diff
@@ -288,6 +288,5 @@ sudo systemctl start crio
 ## Using CRI-O
 
 - Follow this [tutorial](crictl.md) to quickly get started running simple pods and containers.
-- To run quickly with kubernetes, see [our quick kuberentes guide](quick-kubernetes.md).
```

Comment:
So where did this go to?  Is it really of no value?

Reply:
The contents were moved to the README so users could quickly see how to run it, which made the (already very small) tutorial redundent